Taskfile.yml

```diff
@@ -75,51 +68,71 @@ tasks:
       - go build -ldflags "{{.LDFLAGS}}" ../cmd/out
       - go build -ldflags "{{.LDFLAGS}}" ../cmd/hello
 
-  docker-login:
-    silent: true
+  build:copydir:
+    desc: Build copydir (development helper, normally not needed).
+    dir: bin
     cmds:
-      - echo "Logging in to DockerHub"
-      - docker login -u $DOCKER_USERNAME -p $DOCKER_TOKEN 2> /dev/null
+      - go build -ldflags "{{.LDFLAGS}}" ../cmd/copydir
+
+  docker:login:
+    cmds:
+      - echo $DOCKER_TOKEN | docker login -u $DOCKER_USERNAME --password-stdin
     env:
-      DOCKER_TOKEN:
-        sh: echo ${DOCKER_TOKEN:-$(gopass show cogito/docker_token)}
       DOCKER_USERNAME:
-        sh: echo ${DOCKER_USERNAME:-$(gopass show cogito/docker_username)}
+        sh: 'echo {{default "$(gopass show cogito/docker_username)" .DOCKER_USERNAME}}'
+      DOCKER_TOKEN:
+        sh: 'echo {{default "$(gopass show cogito/docker_token)" .DOCKER_TOKEN}}'
 
-  docker-build:
-    deps: [docker-login]
+  docker:build:
     desc: Build the Docker image.
     cmds:
-      - docker build --build-arg BUILD_INFO --tag $DOCKER_IMAGE .
+      - docker build --build-arg BUILD_INFO --tag {{.DOCKER_FULL_NAME}} .
     env:
       BUILD_INFO: "{{.BUILD_INFO}}"
 
-  docker-smoke:
+  docker:smoke:
     desc: Simple smoke test of the Docker image.
     cmds:
-      - echo '{{.INPUT}}' | docker run --rm --interactive cogito /opt/resource/check
+      - echo '{{.INPUT}}' | docker run --rm --interactive {{.DOCKER_FULL_NAME}} /opt/resource/check
       - echo
-      - echo '{{.INPUT}}' | docker run --rm --interactive cogito /opt/resource/in dummy
+      - echo '{{.INPUT}}' | docker run --rm --interactive {{.DOCKER_FULL_NAME}} /opt/resource/in dummy
       - echo
     vars:
       INPUT: '{ "source": { "owner": "foo", "repo": "bar", "access_token": "123"} }'
 
-  docker-push:
-    deps: [docker-login]
+  docker:push:
+    desc: Push the Docker image
     cmds:
-      - docker tag $DOCKER_IMAGE $DOCKER_ORG/$DOCKER_IMAGE:$DOCKER_TAG
-      - echo Pushing $DOCKER_ORG/$DOCKER_IMAGE:$DOCKER_TAG
-      - docker push $DOCKER_ORG/$DOCKER_IMAGE:$DOCKER_TAG
+      - docker push {{.DOCKER_FULL_NAME}}
       - |-
         if [ -n "$DOCKER_LATEST" ]; then
-            docker tag $DOCKER_IMAGE $DOCKER_ORG/$DOCKER_IMAGE:$DOCKER_LATEST
-            echo Pushing $DOCKER_ORG/$DOCKER_IMAGE:$DOCKER_LATEST
-            docker push $DOCKER_ORG/$DOCKER_IMAGE:$DOCKER_LATEST
+            docker tag {{.DOCKER_FULL_NAME}} {{.DOCKER_BASE_NAME}}:latest
+            # FIXME safety while adding GH actions: disable pushing "latest"
+            exit 1
```

Comment:
Will remove the `exit 1` in the PR that adds the capability of making a release (thus pushing to Docker Hub the `latest` tag)